app/controllers/api/v1/article_controller.rb

```diff
@@ -0,0 +1,65 @@
+class API::V1::ArticleController < ApplicationController
+  include ViewPermissionConcern
+
+  before_action :authenticate_user!
```

Comment:
Required call to [devise_token_auth](https://github.com/lynndylanhurley/devise_token_auth), not corrected.